src/DMXSerial2.h

```diff
@@ -133,12 +136,22 @@ extern "C" {
    * @brief Callback function for RDM sensors.
    */
   typedef bool8 (*RDMGetSensorValue)(uint8_t sensorNr, int16_t *value, int16_t *lowestValue, int16_t *highestValue, int16_t *recordedValue);
+
+  /**
+   * @brief Callback function for getting RDM parameters.
+   */
+  typedef bool8 (*RDMGetParameterValue)(uint16_t parameterNr, int8_t *value);
```

Comment:
Parameter index might be clearer, as its not the PID value as I first assumed, although passing in both might be more flexible as people could then write it more like the additional PIDs bit.